test/index-test.js

```diff
@@ -14,18 +14,22 @@ describe('Objects', function() {
         prop2: 2
       })
     })
-    
+
     it('it does not modify the original object, but rather returns a clone with the new data', function() {
       var obj = { prop: 1 }
-    
+
       updateObjectWithKeyAndValue(obj, 'prop2', 2)
 
       expect(obj['prop2']).toBe(undefined)
     })
   })
 
+// UPDATED TO SAY "AND RETURNS THE ENTIRE UPDATED OBJECT"
```

Comment:
@xxDOOMbox if you remove this comment (lines 27-30), I'll merge it on in. Thanks!